src/test/java/io/aiven/kafka/connect/s3/testutils/IndexesToString.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.s3.testutils;
+
+public interface IndexesToString {
```

Comment:
If you feel like it, annotate this with `@FunctionalInterface`. It's mostly a hint to the compiler and doesn't matter much in tests.

Reply:
: +1
- [ ]Annotate with `@FunctionalInterface`